### ISSUE TITLE
Should Opower work for all Evergy customers?

### ISSUE THREAD

**Issue by ourstanley** [tronikos/opower#85]
I was pleased to see my provider Evergy in the list of supported providers, but even though I have authenticated with my Evergy details no power consumption has appeared. Evergy was formed by the merger of Westar with Kansas Power and Light - I’m a former Westar customer. Should this integration work for me, or does it only work for the former Kansas Power and Light customers?

When it was Westar I was able to download data about my consumption in the ‘Blue Button’ format, but that feature was removed post merger and I’ve not found a way to view/analyze my data on anything less than a week/day, wherea previously I could see it for every 15 mins of the day. 

many thanks,

ourstanley

**Comment by tronikos** [comment]
No idea. Try it by running the following:
```
pip install opower
python -m opower 
```

**Comment by ourstanley** [comment]
OK, so after posting this I realized that as well as installing the integration and authenticating to Evergy that I needed to define some entities. So I did that and I seem now to be seeing data - 
<img width="470" alt="Screenshot 2024-07-20 at 2 25 01 PM jpeg" src="https://github.com/user-attachments/assets/9279a8fa-7e0a-4c51-8310-9f5683247d6f">
I’ll monitor this and see how it changes over time. Doesn’t look like I’ll get the consumption by time of day view that came with the Blue Button data, but it might still be useful.
